packages/payload/src/uploads/types.ts

```diff
@@ -82,6 +82,11 @@ export type UploadConfig = {
    * - A function that generates a fully qualified URL for the thumbnail, receives the doc as the only argument.
    **/
   adminThumbnail?: GetAdminThumbnail | string
+  /**
+   * Enables the ability to upload a file from a remote URL instead of selecting a file from your system.
+   * @default false
+   */
+  allowRemoteUpload?: boolean
```

Comment:
Is there any reason this wouldn't be enabled by default (and also sanitized as true)?

Reply:
I suppose there would be no harm to default it to true